src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -554,6 +574,30 @@ class CollapsibleCourseRun extends React.Component {
           extraInput={{ onInvalid: this.openCollapsible }}
           disabled={disabled}
         />
+        {hasExternalKey &&
+          <Field
+            name={`${courseId}.external_key`}
+            component={RenderInputTextField}
+            type="text"
+            label={
+              <FieldLabel
+                id={`${courseId}.external_key.label`}
+                text="Institution Course ID"
+                helpText={
+                  <div>
+                    <p>
+                      Only fill in this field if you are planning to integrate with your
+                      Institution&apos;s systems through edX registrar service.
```

Comment:
`&apos` is the straight apostrophe - just use the `’` character (curly apostrophe) directly maybe?